docs/documentation/install/requirements.md

```diff
@@ -2,7 +2,7 @@
 
 The GOV.UK Prototype Kit runs on Mac, Windows and Linux. At a minimum you’ll need Node.js (install instructions below) and a web browser.
 
-If you're using an M1 Mac ([certain Macs launched in 2020 and later](https://en.wikipedia.org/wiki/Apple_M1#Products_that_include_the_Apple_M1)), you might experience issues when you run the Prototype Kit. To get support, please [contact the Design System team](https://design-system.service.gov.uk/get-in-touch/).
+If you're using an M1 Mac ([certain Macs launched in 2020 and later](https://en.wikipedia.org/wiki/Apple_M1#Products_that_include_the_Apple_M1)), you might experience issues when you run the Prototype Kit. To get support, [contact the Prototype team](https://design-system.service.gov.uk/get-in-touch/).
```

Comment:
Is https://design-system.service.gov.uk/get-in-touch/ best place to get contact details for the prototype team from? It doesn't say anything about the prototypes on it 🤔

Reply:
Good point. We don't have a dedicated contact page on the Kit site, but there is a page to link to with the information.